.github/actions/javascript/getArtifactInfo/getArtifactInfo.ts

```diff
@@ -1,13 +1,16 @@
-const _ = require('underscore');
-const core = require('@actions/core');
-const GithubUtils = require('../../../libs/GithubUtils');
+import * as core from '@actions/core';
+import type {components as OctokitComponents} from '@octokit/openapi-types/types';
+import GithubUtils from '../../../libs/GithubUtils';
 
-const run = function () {
+type OctokitArtifact = OctokitComponents['schemas']['artifact'];
+
+const run = function (): Promise<void> {
     const artifactName = core.getInput('ARTIFACT_NAME', {required: true});
 
+    // TODO: remove type casting once GithubUtils (https://github.com/Expensify/App/pull/38280) is migrated to TypeScript
```

Comment:
Can't we define our own typings for `GithubUtils` in the meanwhile?

Reply:
Not sure what you mean, could you provide more details? 🙏

Reply:
Are you familiar with ambient typings and `.d.ts` files?

In general, typings can be provided by any party. Either the original third-party library itself (let's say `foo`), another third-party library (like `@types/foo`), or the project that depends on `foo` itself.

I'm not sure if there's any difficulty with using ambient typings with GitHub actions specifically, I only used them in "typical" TypeScript projects, but I would assume it's possible.

We can give up if this becomes too time-consuming, but I'd give it a try.

Reply:
Sure, I did it for some other modules, great idea! Actually GithubUtils PR got merged a couple hours ago so the problem is not valid anymore!